notebooks/wav2lip/wav2lip.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>not sure that "novel" word is suitable here (model released 3 years ago)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Fixed</p>